### ISSUE TITLE
I cannot import "Spreadsheet"

### ISSUE THREAD

**Issue by Tas-sos** [gephi/gephi#1036]
I open the Gephi 0.8.1 (in Ubuntu OS 14.04 , 32 - bit ), create a new project and then  go to the tab "Data Laboratory", in "Data Table" there I press the "Import Spreadsheet" but does nothing :( 

Below I get the following error:

![](http://i1222.photobucket.com/albums/dd487/learn1000/gephi-error.png)

![](http://i1222.photobucket.com/albums/dd487/learn1000/gephi-error1.png)

![](http://i1222.photobucket.com/albums/dd487/learn1000/gephi-error2.png)

```
java.lang.ClassNotFoundException: org.netbeans.validation.api.ui.ValidationPanel
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
Caused: java.lang.ClassNotFoundException: org.netbeans.validation.api.ui.ValidationPanel starting from ModuleCL@59eead[org.gephi.datalab.plugin] with possible defining loaders [ModuleCL@1731def[org.gephi.lib.validation]] and declared parents [ModuleCL@156f8b9[org.gephi.datalab.api], ModuleCL@17e1a9e[org.gephi.data.attributes.api], ModuleCL@b7501b[org.openide.dialogs], ModuleCL@58825[org.gephi.lib.javacsv], ModuleCL@b24e3f[org.gephi.ui.utils], ModuleCL@1540d0c[org.openide.windows], ModuleCL@7ff40e[org.jdesktop.swingx], ModuleCL@1731def[org.gephi.lib.validation], ModuleCL@784abe[org.gephi.tools.api], org.netbeans.MainImpl$BootClassLoader@e77ca4, ...5 more]
	at org.netbeans.ProxyClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
Caused: java.lang.NoClassDefFoundError: org/netbeans/validation/api/ui/ValidationPanel
	at org.gephi.datalab.plugin.manipulators.general.ui.ImportCSVUIWizardAction.getPanels(Unknown Source)
	at org.gephi.datalab.plugin.manipulators.general.ui.ImportCSVUIWizardAction.performAction(Unknown Source)
	at org.gephi.datalab.plugin.manipulators.general.ImportCSV.execute(Unknown Source)
[catch] at org.gephi.datalab.api.DataLaboratoryHelper$5.run(Unknown Source)

```

What can I do to fix it ??

Thanks in advance. :)

( Sorry I posted my problem twice, I did not want. I'm new here in github community. :/ )

**Comment by eduramiba** [comment]
Does this work for you in 0.8.2?

**Comment by mbastian** [comment]
Closing as I think this has been fixed in 0.8.2 already.
